deploy/docker/production/entrypoint_deployment.bash

```diff
@@ -9,11 +9,11 @@ poetry run python -m assume_identity
 # Clear db/es on fourfront-mastertest if we run an "initial" deploy
 # Do nothing on other environments
 if [ -n "${INITIAL_DEPLOYMENT}" ]; then
-  poetry run clear-db-es-contents production.ini --app-name app --env fourfront-mastertest
+  poetry run clear-db-es-contents production.ini --app-name app --env fourfront_mastertest
```

Comment:
Should this also work by environment variable with the env name in a variable? Is it already in `ENV_NAME`?